static/src/stylesheets/module/identity/_wizard.scss

```diff
@@ -22,3 +49,25 @@
         display: none;
     }
 }
+
+@keyframes manage-account-wizard__fadeLeft {
+    from {
+        transform: translateX(0);
+        opacity:1;
+    }
+    to {
+        transform: translateX(-5em);
```

Comment:
We try to use `px` where applicable, because they are transpiled into `rem` during the build. Is there a reason why you are using `em` here?

Reply:
not really just habit